lib/ui/painting.dart

```diff
@@ -4452,6 +4452,21 @@ enum TileMode {
   decal,
 }
 
+Float32List _encodeWideColorList(List<Color> colors) {
+  final int colorCount = colors.length;
+  final Float32List result = Float32List(colorCount * 4);
+  for (int i = 0; i < colorCount; ++i) {
```

Comment:
please use `i++`  or `i += 1`. There just isn't any different between pre/post increment in dart and I'd prefer we stay consistent.

Reply:
done